run-linux.sh

```diff
@@ -52,9 +52,10 @@ else
 fi
 
 echo "::group::Installing Apptainer ${v}"
-deb="apptainer_${v/v/}_amd64.deb"
-sudo wget --tries 5 --quiet --timestamping --output-document /var/cache/apt/archives/${deb} https://github.com/apptainer/apptainer/releases/download/${v}/${deb}
-sudo apt-get -q -y install /var/cache/apt/archives/${deb}
+for deb in "apptainer_${v/v/}_amd64.deb" "apptainer-suid_${v/v/}_amd64.deb"; do
```

Comment:
Do you understand why both are needed?

Reply:
apptainer-suid explicitly depends on apptainer, it reports.